public/index.html

```diff
@@ -82,8 +111,10 @@ <h2>PayJoin here to open these channels</h2>
 
 		</main>
 		<script type="text/javascript">
-			function add_channel() {
-				let channels = document.getElementById("channels");
+			let recommendations;
+			var channels = 1;
```

Comment:
think we can delete this `channels = 1`

Reply:
Wait we can remove the channels var entirely?